docs/endpoints/security_posture.rst

```diff
@@ -0,0 +1,144 @@
+In addition to the identity mapping access control, administrators may also use Globus
+authentication policies to narrow which identities can even send tasks to a MEP.  An
+authentication policy can enforce details such as that a user has an identity from a
+specific domain or has authenticated with the Globus Auth recently.  Refer to the
+`Authentication Policies documentation`_ for more background and specifics on what
+Globus authentication policies can do and how they fit in to a site's security posture.
+
+Reference :ref:`MEP § Authentication Policies <auth-policies>` for more information.
+
+
+Function Allow Listing
+----------------------
+
+Administrators can narrow MEP usage by limiting what functions may be requested by
+tasks.  The web-service will reject any submissions that request functions not in the
+MEP's configured ``allowed_functions`` list, and user endpoint processes will again
+verify each task against the same list |nbsp| --- |nbsp| a check at the web-service and
+a check on-site.
+
+Please reference :ref:`MEP § Function Allow Listing <function-allowlist>` for more
+detailed information.
+
+
+Function Argument Serialization
+-------------------------------
+
+Administrators may also fine-tune how task arguments are deserialised by user
+endpoints.  While callables (such as functions) must be serialized by |dill|_ (an
+extension of Python's native |pickle|_ serializer), arguments to functions may be
+restricted to the safer |JSONData| serializer.  This limits what arguments functions
+may receive |nbsp| --- |nbsp| for example, a callable like another function could not
+be passed |nbsp| --- |nbsp| but is safe from arbitrary code execution during
+deserialization.
+
+See :ref:`Endpoint § Restricting Submission Serialization Methods
+<restrict-submission-serialization-methods>` for more information.
+
+
+.. _posture-ha:
+
+High-Assurance (HA)
+-------------------
+
+Endpoints may be designated High-Assurance (HA), which enact a couple of behavior
+differences:
+
+- audit logging is enabled, and (until configured otherwise) logs task events to
+  ``audit.log`` in the endpoint directory
+- enable registration of HA functions with the endpoint; HA functions may only run on
+  the HA endpoint to which they were registered
+- SDK uses of HA functions will similarly incur the HA policy requirements
+- HA functions are cleared from all Globus-related storage (e.g., function name,
+  definition, description) after 3 months of inactivity.
+
+Reference :ref:`MEP § High-Assurance <high-assurance-mep>` for details.
+
+
+Multiple Administrators of a MEP
+--------------------------------
+
+MEPs associated with a subscription may be administered by multiple Globus Auth
+identities.  The identities are stated in the ``admins`` key of the ``config.yaml``:
+
+.. code-block:: yaml
+   :caption: ``config.yaml``
+
+   multi_user: true
+   subscription_id: 600ba9ac-ef16-4387-30ad-60c6cc3a6853
+   admins:
+     # Peter Gibbons (software engineer)
```

Comment:
Are these real people? I'd pick some obviously fictitious names (and fake UUIDs).

My beloved Pine email client used Flintstones characters for its examples.

Reply:
🤦